go.mod

```diff
@@ -1,8 +1,9 @@
-module github.com/IBM/sarama
+module github.com/elastic/sarama
 
 go 1.19
 
 require (
+	github.com/IBM/sarama v1.43.3
```

Comment:
It makes me uncomfortable that upstream is still a dependency, but it looks like this is just referenced in tests and such.